homeassistant/components/husqvarna_automower/const.py

```diff
@@ -1,6 +1,7 @@
 """The constants for the Husqvarna Automower integration."""
 
 DOMAIN = "husqvarna_automower"
+EXECUTION_TIME = 5
```

Comment:
```suggestion
EXECUTION_TIME_DELAY = 5
```

So it clear that it's a delay